create_items.rb

```diff
@@ -0,0 +1,85 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require_relative './list_items'
+
+class Create
+  def initialize(people, books, rentals)
+    @books = books
+    @people = people
+    @rentals = rentals
+    @list = List.new(people, books, rentals)
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [input the number]'
+    select_person = gets.chomp.to_i
+    case select_person
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'invalid input. Input either 1 or 2 according to your choice'
+    end
+  end
```

Comment:
- [ ] A number of your methods (all create methods defined in the _create_items_ file) do not adhere to the single responsibility rule. Kindly ensure that *all* requests for input and input validation are placed in the `main.rb` file and/or classes/methods whose responsibilities are to request or validate input.` I urge you to kindly read further on the same via [this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)